### PR TITLE
Ecosystem Section

### LINKED ISSUE #55: Ecosystem (Partners) Section
- Merging Branch: `developer-portal`
- `src/components/Sections/Partners.svelte`
- Figma: https://www.figma.com/design/DVjqjKrUWwFzgZzXCsVdGv/Frequency_Marketing-Site_?node-id=459-50&t=mAJPgDSEapUaVyAe-1

Includes
- [x] Mobile
- [x] Tablet
- [x] Desktop

NOTE: Only show MeWe and Soar (Others are examples only)

REMEMBER: These images are only to help you orient to the figma file. Figma is the source of truth

Image

Image

Image

### PR DESCRIPTION
**Pull request by wilwade**
# Description

Adds the Ecosystem section to the home page

- Closes #55 

## Screenshot(s)
Screen Shot 2024-11-12 at 17 05 15 (1)
image (3)
image (2)
image (1)

### CODE REVIEW

File: src/components/Header/Header.svelte [hunk 1]
```diff
@@ -8,6 +8,7 @@
     { label: 'About', href: base + '/#about', viewportHighlightId: 'about' },
     { label: 'Users', href: base + '/#users', viewportHighlightId: 'users' },
     { label: 'Developers', href: base + '/#developers', viewportHighlightId: 'developers' },
+    { label: 'Ecosystem', href: base + '/#ecosystem', viewportHighlightId: 'ecosystem' },
```
Comment (wilwade) [inline src/components/Header/Header.svelte:11]: Added a new header item for this as requested

**Review by JoeCap08055 — COMMENTED** [review]
Images appear broken in the PR preview, but it might be a GH thing; they work when I pull the branch locally. Also in the PR preview, the "Companies in the Frequency Ecosystem" title is the wrong size, but appears correct when I run locally.

"The Peoples Bid" image has the text chopped off; looks like it's the actual SVG that's incorrect.

File: src/components/Sections/Ecosystem.svelte [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+  import { Button, Typography } from '@frequency-chain/style-guide';
+  import SectionWrapper from './SectionWrapper.svelte';
+  import TicTokTag from '../TikTokTag.svelte';
```
Comment (shannonwells) [inline src/components/Sections/Ecosystem.svelte:4]:
```suggestion
  import TikTokTag from '../TikTokTag.svelte';
```

File: src/components/Sections/Ecosystem.svelte [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          around shared values, driving social impact, creativity, and growth.
+        </p>
+        <TicTokTag class="my-6 max-w-[300px] self-center md:self-start lg:hidden" />
+        <div class="my-6 self-center md:self-start">
+          <Button size="full" class="max-w-[400px]" href="https://thepeoplesbid.com/"
+            >Learn About the People's Bid</Button
+          >
+        </div>
+      </div>
+      <TicTokTag class="hidden w-full lg:flex lg:w-1/3" />
```
Comment (shannonwells) [inline src/components/Sections/Ecosystem.svelte:36]:
```suggestion
      <TikTokTag class="hidden w-full lg:flex lg:w-1/3" />
```

**Comment by saraswatpuneet** [comment]
Opened in mobile browser and "The People Bid" D seems to step out 

Screenshot_20241112-170637

**Review by shannonwells — APPROVED** [review]
Aside from the typo, I don't have anything to add and I wouldn't do a respin just for that.  Tried it in Chrome and Firefox at mobile, tablet and desktop widths.

**Comment by wilwade** [comment]
> Opened in mobile browser and "The People Bid" D seems to step out

@saraswatpuneet Fixed. Font issue with the svg and just converted it to a path for now.

**Pull request merged**